index.html

```diff
@@ -6,47 +6,174 @@
-    <link rel="stylesheet" href="style.css">
-    </head>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+</head>
 <body>
     <header>
-    <div class="pure-nav-list">
-            <h4 class="important"><a href="#">Ayodele</a></h4>
-            <nav>
-                <div class="menu1">
-                    <div class="menu"></div>
-                    <div class="menu"></div>
-                    <div class="menu"></div>
-                </div>
-            </nav>
-            </div>
-            
-        </header>
-
-        <main>
-    <section class="section-headline">
-    <div class="main-text Developer">
-    <div class="introduction">
-    <h1>Hey There. <br> I’m Ayodele</h1>
-    <h3>I’am a Software Developer</h3>
-    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-    
+    <a href="#"> Ayodele </a>
+    <nav class="toggle">
+        <ul>
+        <li><i class="fas fa-bars"></i></li>
+        </ul>
+    </nav>
+    </header>
+    <section class="content">
+        <h1>Hey There. <br> I’m Ayodele</h1>
+        <h3>I’am a Software Developer</h3>
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <div class="item"><img src="img_social.png" alt="icons"></div>
+    <div id="background"></div>
+    </section>
+
+    <section class="picture">
+        <div class="work-section">
+            <h4>My Recent Works</h4>
+        </div>
+
+        <hr class="solid"/>
+
+    <div class="pictures"></div>
+    <p id="post">Multiple-Post Stories</p>
+    <p class="daily">
+        A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard dummy text ever since 1500s,
+        when an unknown printer took a dummy text.
+    </p>
+    <div class="grid">
+    <ul class="grid-container">
+        <li class="grid-item">css</li>
+        <li class="grid-item">html</li>
+        <li class="grid-item">bootstrap</li>
+        <li class="grid-item">Ruby</li>
+    </ul>
+    </div>
+    <button id="button" type="submit">See Project</button>
+    <br>
+    <div class="bottom">
+    <a href="#"><img src="image.png" alt=""><div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"><li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn active" type="submit">See Project</button></div>
```

Comment:
- [ ] You have done a good job with the project cards but since this image is used as background I would recommend moving this to your CSS file for better practices :wink:
- [ ] Also, the background image is showing until the middle of the card. Maybe the suggestions above could help fix this issue :pushpin: :smiley: 

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144653827-f25c93be-23b7-4cf7-93dd-d4e9373f93fd.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144653862-d30c7f40-0a94-49f8-8dc5-b6b9ad5776bb.png)

Reply:
Resolved accordingly. Thanks.